src/main/java/software/coley/android/xml/AndroidResourceProvider.java

```diff
@@ -61,4 +61,15 @@ public interface AndroidResourceProvider {
 	 */
 	@Nullable
 	String getResEnumName(String resName, long value);
+
+	/**
+	 * @param type
+	 * 		type of resource "attr", "style", "drawable", etc.
+	 * @param resId
+	 * 		Resource ID in the {@code XmlResourceMapChunk}
```

Comment:
Can use `@link` over `@code`

Reply:
Will do I removed this change as its overkill used a simpler solution